openlibrary/templates/type/author/view.html

```diff
@@ -175,10 +179,10 @@ <h2 class="collapse">
 
         $def render_subjects(label, subjects, prefix):
             $if subjects:
-                <div class="section">
+                <div class="section" itemscope itemtype="https://schema.org/CreativeWork">
                     <h6 class="collapse black uppercase">$label</h6>
                     $for subject, count in subjects:
-                        <a href="/subjects/$prefix$subject.lower().replace(' ', '_')">$subject</a>$cond(not loop.last, ",", "")
+                        <a itemprop="about" href="/subjects/$prefix$subject.lower().replace(' ', '_')">$subject</a>$cond(not loop.last, ",", "")
```

Comment:
Having `CreativeWork` here doesn't seem correct; the closes thing looks like having `itemprop="knowsAbout"`. So let's remove the itemtype CreativeWork, and change the `itemprop="about"` to `knowsAbout` (See https://schema.org/Person )